source/ibl_sampler.js

```diff
@@ -98,13 +97,33 @@ class iblSampler
                 texture.data[dst+0] = Math.min((image.dataFloat[src+0])*255, 255);
                 texture.data[dst+1] = Math.min((image.dataFloat[src+1])*255, 255);
                 texture.data[dst+2] = Math.min((image.dataFloat[src+2])*255, 255);
-                texture.data[dst+3] = 0;
+                texture.data[dst+3] = 1.0;
```

Comment:
Assuming that `texture.data` is an array of bytes, the alpha value should be `255`.